src/main/java/mate/academy/service/AuthenticationServiceImpl.java

```diff
@@ -16,7 +20,7 @@ public User login(String login, String password) throws AuthenticationException
 
     private User findByLogin(String login) {
         User user = new User(login, "1234");
-        // this user identifier should be set by DB. We will use dummy data for this example
+        LOGGER.info("FindByLogin method was called. Login = {}", login);
```

Comment:
It is not common practive to write to log what method were called. Better to log what happens, for example Process of searching by login started or something